packages/odyssey-icons/figma/complete.svg

```diff
@@ -1,3 +1,3 @@
 <svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path fill-rule="evenodd" clip-rule="evenodd" d="M6.45774 9.80035L10.7502 1.63976C11.0538 1.06275 11.7594 0.83514 12.3396 1.12712C12.9444 1.43149 13.1777 2.18012 12.8539 2.77761L7.61969 12.4359C7.296 13.0331 6.51148 13.1852 5.99056 12.7516L1.41834 8.94591C0.93322 8.54213 0.859438 7.8214 1.25256 7.32654C1.6392 6.83983 2.33868 6.74723 2.83722 7.11676L6.45774 9.80035Z" fill="#212126"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M7 13C10.3137 13 13 10.3137 13 7C13 3.68629 10.3137 1 7 1C3.68629 1 1 3.68629 1 7C1 10.3137 3.68629 13 7 13ZM9.50016 3.42651L6.63849 8.8669L4.22481 7.07784C3.89245 6.83149 3.42613 6.89322 3.16837 7.21769C2.90629 7.5476 2.95548 8.02808 3.27889 8.29728L6.32704 10.8344C6.67432 11.1235 7.19733 11.0221 7.41313 10.6239L10.9026 4.18507C11.1185 3.78675 10.963 3.28766 10.5598 3.08475C10.173 2.89009 9.70251 3.04183 9.50016 3.42651Z" fill="#212126"/>
 </svg>
```

Comment:
Consider extracting these svg path definitions into a single source of truth. This PR has them reproduced as markdown, vue template and here as svg.

Reply:
@jeffbelser-okta the plan is to have a single Icon component that refers to the odyssey-icons package as the canonical source. Until we have that, we're stuck copy-pasting from that directory throughout the docs.

In the future, I assume Toast will have Icon as a child and pass it the appropriate prop based on the Toast variant.

@arnoldsandoval-okta I think we should remove the paths from the Vue Toast and use conditional SVGs instead. That way, we're at least treating them like content instead of something specific to Toast.